setup.cfg

```diff
@@ -1,5 +1,5 @@
 [pycodestyle]
 count=1
-select = W1
+select = W1, W293, E6, E8, E9
```

Comment:
What's E6, E8 ?

Reply:
E6 and E8 is just a reminder, no test case. 
I had test in my local machine the case from E1 to E9 and in this cases I got no Error back.